packages/plugin-e2e/src/models/DataSourceConfigPage.ts

```diff
@@ -28,9 +29,10 @@ export class DataSourceConfigPage extends GrafanaPage {
    *
    * Optionally, you can skip waiting for the response by passing in { skipWaitForResponse: true } as the options parameter
    */
-  async saveAndTest(options?: TriggerQueryOptions) {
+  async saveAndTest(options?: TriggerQueryOptions): Promise<Response> {
```

Comment:
There's more than a  type fixing here. It is also adding an additional waitForResponse that was not there before. is this intended?

Reply:
Yes this will resolve a playwright response. Looks a bit weird, but with an empty string it resolves immediately. 

The alternative with be to introduce a new method - `saveAndTestWithoutWaitingForResponse`, but I think the approach in this PR is better.

Reply:
Good. I was just making sure since your PR title and description only talked about types and this is actually changing business logic.